doc/rados/operations/health-checks.rst

```diff
@@ -153,6 +153,24 @@ To adjust the warning threshold, run the following command:
 
    ceph config set global mon_data_size_warn <size>
 
+MON_NETSPLIT
+____________
+
+A network partition has occurred among Ceph Monitors. This health check is
+raised when one or more monitors detect that at least two Ceph Monitors have
+lost connectivity or reachability, based on their individual ``connection score``,
```

Comment:
I would take out the double backticks, and make `scores` plural to agree with "their" and "Monitors"